content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -195,6 +197,8 @@ To encrypt the data:
              timeout: 3s
    ```
 
+Setting `--encryption-provider-config-automatic-reload` to `true` collapses all health checks to a single health check endpoint. Individual health checks are only available when KMS v1 providers are in used and encryption config is not auto-reloaded.
```

Comment:
Provide a table that enumerates all possible configs with what healthz checks are present.

i.e. with + without reloads vs. kms v1 only, kms v2 only, kms v1+v2, no kms